packages/server-wallet/src/__test-with-peers__/create-and-close-channel/ledger-funding.test.ts

```diff
@@ -867,7 +871,7 @@ describe('Automatic channel syncing on successive API calls', () => {
     await DBAdmin.truncateDatabase(bWalletConfig, tablesUsingLedgerChannels);
   });
 
-  it('can fund two channels in a situation where first proposal is dropped', async () => {
+  it.skip('can fund two channels in a situation where first proposal is dropped', async () => {
```

Comment:
Should we remove this test?

Reply:
Yeah, probably. I wasn't sure what to do with this - the old ledger algorithm did a lot of extra ("pessimistic") state communication, so it could handle cases like these. The new ledger algorithm doesn't do this - it just relies on sync states being called to handle the case where things have stalled. So the new ledger algorithm fails this test, but that's probably ok?

Reply:
Yep, I think it is totally fine to not handle the same edge cases with the new ledger funding algorithm.